ethcore/res/ethereum/olympic.json

```diff
@@ -17,7 +17,7 @@
 		"accountStartNonce": "0x00",
 		"maximumExtraDataSize": "0x0400",
 		"minGasLimit": "125000",
-		"networkID" : "0x0",
+		"networkID" : "0xf0",
```

Comment:
Should be reverted

Reply:
Yeah sorry I missed this. This line should indeed be reverted and fixed (or we may just consider removing olympics support).